target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -87,13 +87,29 @@ define Device/tplink_archer-c6-v2
   $(Device/tplink-safeloader-uimage)
   ATH_SOC := qca9563
   IMAGE_SIZE := 7808k
+  DEVICE_VENDOR := TP-Link
```

Comment:
DEVICE_VENDOR is inherited from $(Device/tplink-safeloader-uimage).
So, you can remove it here. (But you still require DEVICE_ALT0_VENDOR below!)